system-tests/services/_scripts/external-volumes/volclean.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
```

Comment:
While it's a valid point, I think we should merge this as is. Let's just make sure we put attribution at the top of the file and the authors credentials (in this case Lucas - who works at Mesosphere). That way we know who to contact.